src/patterns/components/highlight-board/react/info/default.hbs

```diff
@@ -0,0 +1,183 @@
+<p class="sprk-u-mbm">See below for available customization options:</p>
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        children
+      </td>
+      <td>
+        Any renderable element(s)
```

Comment:
```suggestion
        Any renderable element(s).
```

Reply:
removed this whole section - React component doesn't use the `children` property.